README.md

```diff
@@ -47,6 +47,11 @@ Windows         | _bazel build --platforms=@io_bazel_rules_go//go/toolchain:wind
 
 For additional target platform names, see the per-Go release definitions [here](https://github.com/bazelbuild/rules_go/blob/master/go/private/sdk_list.bzl#L21-L31) in the _rules_go_ Bazel package.
 
+Additionally if files were moved around, you may need to run the following command to update Bazel files:
```

Comment:
We do mention this point below in the "Keeping the Bazel files up to date" section. I just want to confirm that you intended to duplicate this point here.

Reply:
I added this because I was a bit confused before I figured out the gazelle thing. I didn't really think about the other section. I changed the text so that it references the other section.